src/modalities/__main__.py

```diff
@@ -138,6 +142,41 @@ def entry_point_create_packed_data(src_path, dst_path, index_path, tokenizer_typ
     generator.run(dst_path)
 
 
+# FIXME
```

Comment:
Does this FIXME still need to be addressed? What is it about?

Reply:
@lllAlexanderlll @mrudat-iais Do you know who put this line here? What should be fixed?